demonstrations/tutorial_qnn_multivariate_regression.py

```diff
@@ -0,0 +1,253 @@
+# .. math:: S(\vec{x}) = e^{-i \frac{1}{2} Z x_1} \otimes e^{i \frac{1}{2} Z x_2} = R_z(x_1) \otimes R_z(x_2).
+
+# Specifically, we define the :math:`S(\vec{x})` operator using the :class:`~.pennylane.templates.AngleEmbedding` function
+
+def S(x):
+    qml.AngleEmbedding( x, wires=[0,1],rotation='Z')
+
+######################################################################
+# For the :math:`W(\vec{\theta})` operator, we will use an ansatz that is available in PennyLane, called :class:`~pennylane..templates.StronglyEntanglingLayers`.
+
+def W(params):
+    qml.StronglyEntanglingLayers(params, wires=[0,1])
+
+######################################################################
+# Now we will build the circuit in PennyLane by alternating layers of :math:`W(\vec{\theta})` and :math:`S(x)` layers. The output that we are getting from the circuit is the expectation value of the :math:`Z` operator on both qubits, which we will compute using PennyLane's expval function.
+
+@qml.qnode(dev,interface="jax")
+def quantum_neural_network(params, x):
+    layers=len(params[:,0,0])
+    for i in range(layers):
+      W(params[i,:,:].reshape(1,len(params[0,:,0]),len(params[0,0,:])))
+      S(x)
+    W(params[-1,:,:].reshape(1,len(params[0,:,0]),len(params[0,0,:])))
+
+    return qml.expval(qml.PauliZ(wires=0)@qml.PauliZ(wires=1))
+
+######################################################################
+# The function we will be fitting is :math:`f(x_1, x_2) = \frac{1}{2} \left( x_1^2 + x_2^2 \right)`, which we will define as target_function:
+
+def target_function(x):
+    f=1/2*(x[0]**2+x[1]**2)
+    return f
+
+######################################################################
+# Now we will specify the range of :math:`x_1` and :math:`x_2` values and store those values in an input data vector. We are fitting the function for :math:`x_1, x_2 \in [-1, 1]`, using 30 evenly-spaced samples for each variable.
+
+x1_min=-1
+x1_max=1
+x2_min=-1
+x2_max=1
+num_samples=30
+
+######################################################################
+# Now we build the training data with the exact target function :math:`f(x_1, x_2)`.
+
+x1_train=np.linspace(x1_min,x1_max, num_samples)
+x2_train=np.linspace(x2_min,x2_max, num_samples)
+x1_mesh,x2_mesh=np.meshgrid(x1_train, x2_train)
+
+######################################################################
+# We define x_train, y_train
+x_train=np.stack((x1_mesh.flatten(), x2_mesh.flatten()), axis=1)
+y_train = np.array(np.real(target_function([x1_mesh,x2_mesh])).reshape(-1,1))
+
+######################################################################
+# What do we do with the output from the circuit?
+# ------------------------------------------
+#
+# We want to optimize the circuit above so that the :math:`Z` expectation value 
```

Comment:
It might be a matter of taste, but for me "Z expectation value" sounds weird. I think one of these could work:
- expectation value in the computational basis
-  expectation value of $Z \otimes Z$